kong/db/strategies/postgres/connector.lua

```diff
@@ -298,7 +295,7 @@ function _mt:init()
   local res, err = self:query("SHOW server_version_num;")
   local ver = tonumber(res and res[1] and res[1].server_version_num)
   if not ver then
-    return nil, "failed to retrieve PostgreSQL server_version_num: " .. err
+    return nil, "failed to retrieve PostgreSQL server_version_num: " .. err or ""
```

Comment:
Is this way of writing correct? When I test in Luajit, I get this error
```
> do
   local err = nil
   print("failed to retrieve PostgreSQL server_version_num: " .. err or "")
end
>>
stdin:3: attempt to concatenate local 'err' (a nil value)
stack traceback:
	stdin:3: in main chunk
	[C]: at 0x00405340
```

Reply:
Good catch. Looks like `..` has higher priority.